cmd/watcher/main.go

```diff
@@ -50,7 +50,7 @@ import (
 
 const (
 	// Service Account token path. See https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod
-	podTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+	podTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token" //nolint:gosec
```

Comment:
Please add a comment explaining this is not a hard-coded secret:

```suggestion
        // This is a fixed path which does not contain a hard-coded secret or credential
	podTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token" //nolint:gosec
```